src/mapping/config/MappingConfiguration.cpp

```diff
@@ -248,7 +248,7 @@ MappingConfiguration::MappingConfiguration(
   auto attrGeoMultiscaleRadius = XMLAttribute<double>(ATTR_GEOMETRIC_MULTISCALE_RADIUS)
                                      .setDocumentation("Radius of the circular interface between the 1D and 3D participant.");
   auto attrSpreadProfile = XMLAttribute<std::string>(ATTR_GEOMETRIC_MULTISCALE_SPREAD_PROFILE)
```

Comment:
To follow the current convention, let's rename the attribute to `attrGeoMultiscaleSpreadProfile`.